app/api/predict/route.ts

```
@@ -0,0 +1,76 @@
+const predictionData = predictions.dataSync();
+
+    // Extract predicted return and volatility from predictionData
+    // This depends on how your model's output is structured
+    const predicted_return = predictionData[0]; // Example
+    const predicted_volatility = predictionData[1]; // Example
+
+    return NextResponse.json({
+      symbol,
+      predicted_return,
+      predicted_volatility,
+    });
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)